uiowa/uiowa
# [ITS] Service Alerts

Resolves #7035 





# How to test


```
ddev blt ds --site=its.uiowa.edu && ddev drush @its.local uli /admin/structure/taxonomy/manage/alert_categories/add
```

- Add some Alert Categories, such as `Outage, Service Announcement, Maintenance, Degradation of Service`
- Visit `/node/add/alert` and add an alert
- Try adding different versions of known duration alerts (eg Tuesday from 9am - 10pm) or open-ended alerts (eg started at 9am, assumed to be unresolved at this time), and see if the smartdate form makes sense/is intuitive for these.
- Test perms w/content moderation: editors/publishers/webmasters should be able to create and edit alerts, but only publishers/webmasters should be able to publish

```
ddev blt ds --site=facilities.uiowa.edu && ddev drush @facilities.local uli /node/add/alert
```
Check for a clean sync and test the form with an eye toward the change to Dates affected, considering Facilities more often than not works with a specified end time for their maintenance alerts, and compare w/the experience on ITS
https://iowaweb.slack.com/archives/C0164FNN1PV/p1698764954582949
